src/js/utils/dom.js

```diff
@@ -209,3 +209,28 @@ export function nextSibling(element) {
 export function previousSibling(element) {
     return element.previousElementSibling;
 }
+
```

Comment:
Won't this do the trick?
https://github.com/jwplayer/jwplayer/blob/3bef4d3d8dbf2aec3a7fa5df1cabddcb8a9ea7f5/src/js/utils/browser.js#L66

Reply:
Awesome! Definitely prefer this to crawling up the DOM tree.